prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -65,6 +65,21 @@ public async Task<ActionResult> GetEnrollees([FromQuery] EnrolleeSearchOptions s
             }
         }
 
+        // GET: api/Enrollees/1/adjacent
+        /// <summary>
+        /// Gets adjacent next and previous enrollee IDs for a given enrolleeId
+        /// </summary>
+        [HttpGet("{enrolleeId}/adjacent", Name = nameof(GetAdjacentEnrolleeId))]
+        [Authorize(Roles = Roles.PrimeAdministrant)]
```

Comment:
Prime Administrant is not a permissions level Role. It is used almost exclusively for frontend guards/routing. The correct role here would be EnrolleeView

Reply:
Fixed in f103e5c